KlayGE/Plugins/Include/KlayGE/OpenGLES/OGLESRenderView.hpp

```diff
@@ -99,7 +99,6 @@ namespace KlayGE
 
 	private:
 		OGLESTexture1D& texture_1d_;
-		int array_index_;

```

Comment:
Please keep array_index, and use UNREF_PARAM in constructor on it. Texture array will be implemented very soon.